src/main/conversions/v2.26.0/c2_26_0_2019040901.clj

```diff
@@ -0,0 +1,18 @@
+(ns facepalm.c2-26-0-2019040901
+  (:use [kameleon.sql-reader :only [load-sql-file]]))
+
+(def ^:private version
+  "The destination database version"
+  "2.26.0:2019049.01")
```

Comment:
I think this version was meant to be `2.26.0:20190409.01` (missing the `0` before the `9`: `09.01`)

Reply:
Ah, good catch! I'll fix it.

Reply:
This should be fixed now.